.github/workflows/windows-package.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           python setup.py bdist_wheel
       if: ${{ github.event_name == 'release' }}
            python -m pip install twine
+           twine upload  dist/*
```

Comment:
You will have to use the multiline run block to add all these commands together.
See https://github.com/CliMT/climt/blob/87ad69cf3bd510514d263863d4c6effd6ad8e105/.github/workflows/python-package.yml#L47